tests/api_data/dsm_6/core/const_6_core_share.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
+"""DSM 6 SYNO.Core.Share data."""
+
+DSM_6_CORE_SHARE = {
```

Comment:
> I can't figure out the issue with the test, I'll need your help.

Oh sorry, I thought it was the `urlencode` issue.

So, real response to my comment https://github.com/ProtoThis/python-synology/pull/61#discussion_r448379291 :
Indeed, this is exactly what you did by adding 
```
            if SynoShare.API_KEY in url:
                return API_SWITCHER[self.dsm_version]["CORE_SHARE"]
```
to `tests/__init__.py`.

Thanks :relaxed: